index.js

```diff
@@ -230,6 +231,26 @@ function getHighScore() {
   }
 }
 
+function startbgm(){
+  var x = document.getElementById("player");
```

Comment:
Please use `let` or `const` instead of `var`

Reply:
https://incompetech.filmmusic.io/song/3358-amazing-plan

Reply:
i have changed var to let . ignore the typo in the :P